lib/ansible/modules/windows/win_domain.ps1

```diff
@@ -75,7 +75,10 @@ try {
     $forest_context = New-Object -TypeName System.DirectoryServices.ActiveDirectory.DirectoryContext -ArgumentList Forest, $dns_domain_name
     $forest = [System.DirectoryServices.ActiveDirectory.Forest]::GetForest($forest_context)
 } catch [System.DirectoryServices.ActiveDirectory.ActiveDirectoryObjectNotFoundException] {
-} catch [System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException] { }
+    Add-Warning -obj $result -message "Error during forest query initialization: $($_.Exception.Message)"
```

Comment:
No use for this warning, it's normal for this exception to be fired in cases where the forest does not exist.

Reply:
ill change it to out-null then

Reply:
Just keep it empty and ignore the linting error, having `Out-Null` is unecessary.